### PR TITLE
Slightly improve help for sign/verify

### LINKED ISSUE #2875: Need better descriptions for `id` and `groupid` options in `sign` and `verify` commands
## Version of Singularity:

master

I'm not sure what the `--id` and `--groupid` options do in the `sign` and `verify` commands and no documentation or examples are attached to them.

I think something should be done to help clear up what they are to other users, like me 😄

### PR DESCRIPTION
**Pull request by WestleyK**
## Description of the Pull Request (PR):

This PR slightly improves the help for sign and verify.

### This fixes or addresses the following GitHub issues:

 - Fixes #2875

#### Before submitting a PR, make sure you have done the following:

- Read the [Guidelines for Contributing](https://github.com/sylabs/singularity/blob/master/CONTRIBUTING.md), and this PR conforms to the stated requirements.
- Added changes to the [CHANGELOG](https://github.com/sylabs/singularity/blob/master/CHANGELOG.md) if necessary according to the [Contribution Guidelines](https://github.com/sylabs/singularity/blob/master/CONTRIBUTING.md)
- Added tests to validate this PR and tested this PR locally with a `make testall`
- Based this PR against the appropriate branch according to the [Contribution Guidelines](https://github.com/sylabs/singularity/blob/master/CONTRIBUTING.md)
- Added myself as a contributor to the [Contributors File](https://github.com/sylabs/singularity/blob/master/CONTRIBUTORS.md)

Attn: @singularity-maintainers

### CODE REVIEW

File: cmd/internal/cli/sign.go [hunk 1]
```diff
@@ -27,7 +27,7 @@ var signSifGroupIDFlag = cmdline.Flag{
 	DefaultValue: uint32(0),
 	Name:         "group-id",
 	ShortHand:    "g",
-	Usage:        "group ID to be signed",
+	Usage:        "sign a specified group ID (default non)",
```
Comment (dtrudg) [inline cmd/internal/cli/sign.go]: I'm not sure this change really makes it any clearer _what_ the option does. Maybe something like:

"sign over multiple partitions/descriptors in the specified group"

File: cmd/internal/cli/sign.go [hunk 2]
```diff
@@ -47,7 +47,7 @@ var signSifDescSifIDFlag = cmdline.Flag{
 	DefaultValue: uint32(0),
 	Name:         "sif-id",
 	ShortHand:    "i",
-	Usage:        "descriptor ID to be signed (default system-partition)",
+	Usage:        "sign a specified descriptor ID (default system-partition)",
```
Comment (dtrudg) [inline cmd/internal/cli/sign.go]: Ditto...

"sign a single partition/descriptor that has the specified ID" ?

**Review by dtrudg — CHANGES_REQUESTED** [review]
No huge objection, but I think since we are doing changes here we need to take the time here to clarify whether we call things "descriptor" or "partition" throughout the CLI and be consistent / clear.

File: cmd/internal/cli/sign.go (hunk 1 above)
Comment (WestleyK) [reply cmd/internal/cli/sign.go]: I changed it to:

```
sign all partitions in the specified group (default non)
```

Hows that?

**Review by dtrudg — APPROVED** [review]

**Pull request merged**